components/dom_crawler.rst

```diff
@@ -282,6 +282,60 @@ and :phpclass:`DOMNode` objects:
 
         $html = $crawler->html();
 
+Expression Evaluation
+~~~~~~~~~~~~~~~~~~~~~
+
+The ``evaluate()`` method evaluates the given XPath expression.
+The return value depends on if the expression operates on simple values
+(like HTML attributes), or returns a subset of the current document.
+If the expression evaluates to a scalar value, an array of results will be
+returned. If the expression evaluates to a DOM document, a new ``Crawler``
+instance will be returned.
+
+This behavior is best illustrated with examples:
+
+.. code-block:: php
```

Comment:
please transform these 3 lines to `This behaviour is best illustrated with examples::`